src/Command/CodeStudio/CodeStudioWizardCommand.php

```diff
@@ -42,14 +42,33 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     // But, we specifically need an API Token key-pair of Code Studio.
     // So we reauthenticate to be sure we're using the provided credentials.
     $this->reAuthenticate($cloudKey, $cloudSecret, $this->cloudCredentials->getBaseUri(), $this->cloudCredentials->getAccountsUri());
-
-    $phpVersions = [
-      'PHP_version_8.1' => "8.1",
-      'PHP_version_8.2' => "8.2",
+    $phpVersion = NULL;
+    $nodeVersion = NULL;
+    $projectType = [
+      'Drupal_project',
+      'Node_project',
     ];
-    $project = $this->io->choice('Select a PHP version', array_values($phpVersions), $phpVersions['PHP_version_8.1']);
-    $project = array_search($project, $phpVersions, TRUE);
-    $phpVersion = $phpVersions[$project];
+    $projectSelected = $this->io->choice('Select a project type', $projectType, $projectType[0]);
+    echo "Selected project is, $projectSelected";
```

Comment:
Is this is a debugging statement?

Reply:
Removed debugging statement : [60bf6c6](https://github.com/acquia/cli/pull/1683/commits/60bf6c67023bcee873512fae1c88a952174a9347)